models/looksrare/ethereum/looksrare_ethereum_events.sql

```diff
@@ -163,17 +163,20 @@ INNER JOIN {{ source('ethereum','transactions') }} tx ON tx_hash = tx.hash
     AND tx.block_time > '2022-01-01'
     {% endif %}
     {% if is_incremental() %}
-    AND TRY_CAST(date_trunc('DAY', tx.block_time) AS date) = TRY_CAST(date_trunc('DAY', looks_rare.block_time) AS date)
+    AND tx.block_time >= date_trunc("day", now() - interval '1 week')
```

Comment:
I've noticed you often use the `tx.block_time >= date_trunc("day", now() - interval '1 week')` filter

I was wondering why go back one week when we could probably just use something like this instead (which is supposed to take the last timestamp for the previous materialization and not the whole week before today:
`WHERE tx.block_time >= (select max(block_time) from {{ this }})`

Reply:
This needs to be a static value for the filter to be pushed down when fetching the files otherwise it does the filtering after doing a full scan. We could potentially make this shorter as well.